### PR TITLE
Create parachain_onboarding.md

### PR DESCRIPTION
**Pull request by educlerici-zondax**

### CODE REVIEW

File: docs/parachain_onboarding.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+6. **FAQ**
+7. **Support and Resources**
+
+## 1. Prerequisites
+
+Before initiating the onboarding process, ensure you meet the following requirements:
+
+- **Technical Requirements:** Have a fully operational blockchain capable of connecting as a parachain. This includes having a collator setup ready.
+- **Resource Allocation:** Ensure you have the computational and network resources to run a stable collator node.
+- **Paseo Tokens:** Acquire sufficient PAS tokens for staking if required for becoming a parachain on the Paseo Testnet.
```
Comment (al3mart) [inline docs/parachain_onboarding.md:23]:
```suggestion
- **Paseo Tokens:** Acquire sufficient PAS tokens to register your chain's state and code.  The faucet is available for everybody to use as means of getting PAS https://faucet.polkadot.io/paseo.

If having troubles, please reach out at https://matrix.to/#/#paseo-testnet-support:parity.io
```

File: docs/parachain_onboarding.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+## 2. Parachain Registration Process
+
+To register your blockchain as a parachain on the Paseo Testnet:
+
+### 2.1 Application Submission
+- Submit an application through the official Paseo Testnet website. Include technical details of your blockchain, team information, and your objectives for joining the testnet.
```
Comment (al3mart) [inline docs/parachain_onboarding.md:30]: Right now we don't have a website live.
Maybe better point people to open an issue in the support repo.

File: docs/parachain_onboarding.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+### 2.2 Review and Approval
+- The Paseo team will review your application. Approval is based on the readiness of your technology, the potential impact on the Paseo ecosystem, and the availability of parachain slots.
+
+### 2.3 Slot Auctions
```
Comment (al3mart) [inline docs/parachain_onboarding.md:35]: We are not doing auctions. Let's just ask teams to request a slot directly to the team

File: docs/parachain_onboarding.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+## 3. Setting Up Your Parachain
+
+### 3.1 Configuration
+- Configure your blockchain’s collator nodes according to the Paseo network specifications. This includes setting up network endpoints, consensus mechanisms, and synchronization settings.
+
+### 3.2 Genesis State
+- Prepare the genesis state of your parachain. This includes setting the initial state, configuring balances, and deploying initial smart contracts or pallets.
+
+## 4. Connecting Your Parachain to Paseo
```
Comment (al3mart) [inline docs/parachain_onboarding.md:46]: Would remove this entire point in favor of including an example in section 3 on how to run the node, pointing to the paseo spec for easy discoverability

File: docs/parachain_onboarding.md (hunk 2 above)
Comment (educlerici-zondax) [reply docs/parachain_onboarding.md]: updated! can you please check it again?

File: docs/parachain_onboarding.md (hunk 3 above)
Comment (educlerici-zondax) [reply docs/parachain_onboarding.md]: updated! can you please check it again?

File: docs/parachain_onboarding.md (hunk 4 above)
Comment (educlerici-zondax) [reply docs/parachain_onboarding.md:42]: updated! can you please check it again?

**Review by al3mart — APPROVED** [review]

**Pull request merged**